plugins/acpi-dmar/README.md

```diff
@@ -6,3 +6,7 @@ Introduction
 
 This plugin checks if DMA remapping for Thunderbolt devices is available. The
 result will be stored in an security attribute for HSI.
+
+External interface access
+---
```

Comment:
I think the markdown crazies might get upset the --- doesn't extend all the way to the `access`